src/components/home/elements/TokenRecord.vue

```diff
@@ -4,14 +4,19 @@
     <b-row align-v="center">
       <b-col class="record-icon"
              cols="auto">
-        <img src="@/assets/imgs/icons/wallet/ic_token2.svg"
+        <img v-if="isCertified"
+             src="@/assets/imgs/icons/wallet/ic_token1.svg"
```

Comment:
Every certified token has its own logo. We should use method to get its icon here. Create method `getTokenIcon(...)` and use method to show its own logo.